app/page/review/index.tsx

```diff
@@ -1,12 +1,46 @@
 import { useEffect, useState } from "react"
+import browser from "webextension-polyfill"
 import { useHashLocation } from "wouter/use-hash-location"
 
 import { Path } from "~app/path"
 import { useAction, usePendingTransactions } from "~app/storage"
 
 import { TransactionConfirmation } from "./transaction"
 
-function ProfileSwithcher(props: {
+export function Review() {
+  const [, navigate] = useHashLocation()
+  const pendingTxs = usePendingTransactions()
+
+  useEffect(() => {
+    async function redirect() {
+      const [tab] = await browser.tabs.query({
+        active: true,
+        lastFocusedWindow: true
+      })
+      if (tab.url?.includes("notification")) {
```

Comment:
After testing, the following code should be used to prevent the `TypeError: Cannot read properties of undefined (reading 'url')` error:

```ts
if (tab?.url?.includes("notification")) {
```